Libplanet/Blocks/Block.cs

```diff
@@ -174,8 +171,12 @@ public byte[] ToBencodex(bool hash, bool transactionData)
                 );
             foreach (Transaction<T> tx in Transactions)
             {
-                var triples = tx.EvaluateActionsGradually(
-                    Hash, Index, delta, Miner.Value);
+                IEnumerable<ActionEvaluation<T>> triples =
+                    tx.EvaluateActionsGradually(
+                        Hash,
+                        Index,
+                        delta,
+                        Miner.Value);
```

Comment:
Is this just a style adjustment?

Reply:
Yes. and I also convert `var` to explicit type(`IEnumerable<ActionEvaluation<T>>`).

Come to think of it, it would be better if `triples` is renamed to `evaluations`.